### PR TITLE
feat: Add `undo` and `redo` icons

### PR DESCRIPTION
**Pull request by aelfric**
Possible response to [#60](https://github.com/feathericons/feather/issues/60) based on a suggestion to use `refresh-cw` as a starting point.

image
image

### CODE REVIEW

**Comment by ericfennis** [comment]
Awesome @aelfric , nice icons!

**Comment by ericfennis** [comment]
Maybe aligning the lines more to the grid. That will look better on low-res devices.

**Comment by aelfric** [comment]
Re: the suggestion to align to the grid, you mean something like this?

image

I can update them both.

**Comment by locness3** [comment]
Maybe also align the start of the arrow?

**Comment by ericfennis** [comment]
@aelfric yeah exactly. Look better. And I agree with @locness3 maybe start the line in same height as where the arrow ends.

File: icons/redo.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+ <polyline transform="matrix(-1,0,0,1,23,5)" points="1 4 1 10 7 10"/>
+ <path d="m2.5102 14c2.2082-6.2402 10.171-8.0418 14.85-3.36l4.64 4.36"/>
```
Comment (jletey) [inline icons/redo.svg:13]:
```suggestion
  <polyline transform="matrix(-1,0,0,1,23,5)" points="1 4 1 10 7 10"/>
  <path d="m2.5102 14c2.2082-6.2402 10.171-8.0418 14.85-3.36l4.64 4.36"/>
```

File: icons/undo.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+ <polyline transform="translate(.9998 5.9996)" points="1 4 1 10 7 10"/>
+ <path d="M 21.49,15 C 19.2818,8.7598 11.319,6.9582 6.64,11.64 L 2,16"/>
```
Comment (jletey) [inline icons/undo.svg:13]:
```suggestion
  <polyline transform="translate(.9998 5.9996)" points="1 4 1 10 7 10"/>
  <path d="M 21.49,15 C 19.2818,8.7598 11.319,6.9582 6.64,11.64 L 2,16"/>
```

**Comment by aelfric** [comment]
I tried moving the start point.  Thoughts?

image

File: icons/redo.svg (hunk 1 above)
Comment (aelfric) [reply icons/redo.svg]: Fixed the indentation in my third commit.

File: icons/undo.svg (hunk 2 above)
Comment (aelfric) [reply icons/undo.svg]: Fixed the indentation in my third commit.

**Comment by locness3** [comment]
looks kinda weird at 100% size 
Screen Shot 2020-07-05 at 15 56 40

File: icons/redo.svg [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <polyline transform="matrix(-1 0 0 1 23 6)" points="1 4 1 10 7 10"/>
```
Comment (ericfennis) [inline icons/redo.svg:12]:
```suggestion
  <polyline points="22 10 22 16 16 16" />
```

File: icons/undo.svg [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <polyline transform="translate(1 6)" points="1 4 1 10 7 10"/>
```
Comment (ericfennis) [inline icons/undo.svg:12]:
```suggestion
  <polyline points="2 10 2 16 8 16" />
```

File: icons/undo.svg [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  <path d="m22 16c-1.8136-6.5424-9.5593-9.4823-15.36-4.36l-4.64 4.36"/>
```
Comment (ericfennis) [inline icons/undo.svg:13]:
```suggestion
  <path d="m22 16c-1.8136-6.5424-9.5593-9.4823-15.36-4.36l-4.64 4.36" />
```

**Pull request merged**